apps/communications/contacts/test/marionette/ice_test.js

```diff
@@ -13,10 +13,16 @@ marionette('Contacts > ICE contacts', function() {
 
   var testContact = {
     givenName: ['Hello'],
-    tel: [{
-      type: ['mobile'],
-      value: testContactTelNumber
-    }]
```

Comment:
instead of doing this, I think it would be better to add another testContact with two tel numbers and add specific test cases that deal with it.